### PR TITLE
Initialized react-native app

### PR DESCRIPTION
**Pull request by thearavind**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,21 +1,3 @@
-MIT License
```
Comment (ravirajsubramanian) [inline LICENSE]: I think you have changed the license file mistakenly instead of README file.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+language: node_js
+
+node_js:
+  - "node"
+
+script:
+  - yarn test
```
Comment (ravirajsubramanian) [inline .travis.yml]: Leave a blank line at the bottom. It's needed in a few other files.

File: LICENSE (hunk 1 above)
Comment (thearavind) [reply LICENSE]: Reverted back :man_facepalming:

File: .travis.yml (hunk 2 above)
Comment (thearavind) [reply .travis.yml]: Done :+1:

**Pull request merged**